misc/table/src/TFileIter.cxx

```diff
@@ -465,7 +465,7 @@ TKey *TFileIter::NextEventKey(UInt_t eventNumber, UInt_t runNumber, const char *
       if (runNumber != UInt_t(-1)) {
          UInt_t thisRunNumber = thisKey.RunNumber();
          if (thisRunNumber < runNumber) continue;
-         if (thisRunNumber < runNumber) { key = 0; break; }
+         if (thisRunNumber > runNumber) { key = 0; break; }
```

Comment:
@pcanal Could you please comment on this change? I am not sure about it.

Reply:
LGTM